core/relation/status.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package relation
+
+// Status describes the status of a relation.
+type Status string
+
+const (
+	Active  Status = "active"
+	Revoked Status = "revoked"
+)
```

Comment:
Some documentation of what Status is about and what the specific values mean would be helpful. 

I'd also like to know how Life and Status interact. For example, what does it mean for a relation to be Dead and Active. Is that even allowed? What other possible combinations are there?